_posts/blog/2017-07-14-fluo-graduates-from-apache-incubator.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Fluo graduates from the Apache incubator!"
+date: 2017-07-14 11:50:00 +0000
+---
+
+Apache Fluo has graduated from the Apache Incubator to become a Top-Level Project! Graduation signifies Fluo's commitment to the meritocratic process and principles of the Apache Software Foundation. For more information, see [the announcement][post] on the Apache Software Foundation blog.
```

Comment:
Could drop `has`  ... `Apache Fluo graduated`

Apache Incubator could link to http://incubator.apache.org/